config/jobs/kubernetes-sigs/controller-tools/controller-tools-presubmits-master.yaml

```diff
@@ -12,6 +13,13 @@ presubmits:
         command:
         - make
         - test-all
+        resources:
+          limits:
```

Comment:
Do we know if this is sufficient?

Reply:
Based on this discussion https://kubernetes.slack.com/archives/C7J9RP96G/p1685551306721239
the nodes in the EKS cluster have 8 Gi per core. So I would suggest to go with 4 Gi per core for now and then we can discuss reducing this later once we have data (which we can only get from the eks cluster)

So I would go with 24 Gi for now

Reply:
Or let's go with 18 should be easily enough I guess